src/Frontend/Core/Header/Header.php

```diff
@@ -126,7 +126,7 @@ public function __construct(KernelInterface $kernel)
         $this->meta = new MetaCollection();
 
         // add some default CSS files
-        $this->addCSS('/src/Frontend/Core/Layout/Css/screen.css');
+        $this->addCSS('/src/Frontend/Themes/' . Theme::getTheme() . '/Core/Layout/Css/screen.css');
```

Comment:
with this you only scratch your itch, what would be better is a more generic solution, does this file exist in the current theme, ok, use that one, else fall back to the core